serializers/neos_serializer.rb

```diff
@@ -0,0 +1,14 @@
+class NeosSerializer
+  def daily(response)
+    neos = []
```

Comment:
current json response is [{neo1},{neo2},{neo3},{neo4}]
each neo is {id: :id, name: :name, relative_velocity: :relative velocity, lunar_distance: :lunar_distance}